sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_diagnostics/diagnostic_logging.py

```diff
@@ -26,12 +26,21 @@
 _logger.propagate = False
 _logger.setLevel(logging.INFO)
 _DIAGNOSTIC_LOG_PATH = _get_log_path()
+
 _DISTRO_DETECTS_ATTACH = "4100"
+_INFO = "4101"
+_INSTRUMENTATION_SKIPPED = "4101"
+
 _ATTACH_SUCCESS_DISTRO = "4200"
 _ATTACH_SUCCESS_CONFIGURATOR = "4201"
+_INSTRUMENTATION_SUCCEEDED = "4202"
```

Comment:
I know these are arbitrary but does it make sense to group the related codes together (like instrumentation)?

Reply:
Yes but they are grouped by severity instead. For instance, skipping in instrumentation is by design. For instance, it flask would be skipped if the flask library is not installed. But a failing instrumentation is unintentional.